pinot-plugins/pinot-input-format/pinot-csv/src/main/java/org/apache/pinot/plugin/inputformat/csv/CSVRecordReaderConfig.java

```diff
@@ -32,8 +32,8 @@
   private String _header;
   private char _delimiter = DEFAULT_DELIMITER;
   private char _multiValueDelimiter = DEFAULT_MULTI_VALUE_DELIMITER;
-
   private Character _commentMarker;  // Default is null
+  private char _escapeCharacter;
```

Comment:
Should we have a default escape character? Or use `null` as default? Current default will be `'\0'`

Reply:
Yes, the `\0` is a typo, it should either be `null` (which is the default in current master) or `\` (since that is the common escape character in use). From what I see, backward compatibility issues are minimal here, so setting escape character to `\`, but if there are good reasons to continue with `null`, we can do that.